### PR TITLE
Rework safety check on size arguments for when LV doesn't exist

### LINKED ISSUE #3665: #2267 lead to behaviour regression
### Summary

I recently upgraded to 3.5.0, which is past the point #2267 was merged. 

This PR broke some behaviour I have been using for a long time with the lvol module.

I manage a large number of VMs - when they're created from a template with variable disk size, I create an lvol `data` that fills all free space. If, further down the line, I simply expand the disk and run the same ansible config the `data` lvol is expanded to fill the new space. This has worked reliably well for a long time with the lvol module.

With #2267, a check was added to throw an error if an lvol didn't exist, but the size argument had an operator (+/-).
```
            if size_operator is not None:
                module.fail_json(msg="Bad size specification of '%s%s' for creating LV" % (size_operator, size))
```

Whilst I understand why this was added, the change of behaviour now means that I cannot use the single module call I use now.

The workaround I can now think of is:
* Get current lvol info
* If `data` doesn't exist, call lvol with 100%FREE
* If `data` does exist, call lvol with +100%FREE

This is turning a single module call into 3 module calls. Ansible is already slow enough, having to do this arduous work around for a long-running reasonable use case seems excessive.

Would a PR be accepted if I removed this clause? Other than 'user friendliness', is this safety check justified for the lack of functionality? The module now forces you to pass different arguments depending on if the resource exists or not, to get to the same end point. This doesn't seem very user friendly or idempotent.

### Issue Type

Bug Report

### Component Name

community/general/plugins/modules/lvol.py

### Ansible Version

```console (paste below)
$ ansible --version
ansible [core 2.11.3] 
  config file = /home/jr/ansible/ansible.cfg
  configured module search path = ['/home/jrs/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/jr/ansible/venv/lib64/python3.8/site-packages/ansible
  ansible collection location = /home/jr/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/jr/ansible/venv/bin/ansible
  python version = 3.8.8 (default, Aug 25 2021, 16:13:02) [GCC 8.5.0 20210514 (Red Hat 8.5.0-3)]
  jinja version = 3.0.1
  libyaml = True

```

### Community.general Version

```console (paste below)
$ ansible-galaxy collection list community.general
3.5.0
```

### Configuration

```console (paste below)
$ ansible-config dump --only-changed

```

### OS / Environment

_No response_

### Steps to Reproduce

```yaml (paste below)
- lvol:
    vg: system
    lv: data
    size: +100%FREE
```

### Expected Results

An lvol to be created, or expanded, to fill all available space

### Actual Results

```console (paste below)

failed: [hostname] (item={'vg': 'system', 'lv': 'data', 'size': '+100%FREE'}) => {
    "ansible_loop_var": "item",
    "changed": false,
    "item": {
        "lv": "data",
        "size": "+100%FREE",
        "vg": "system"
    }
}

MSG:

Bad size specification of '+100%FREE' for creating LV
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by jake2184**
##### SUMMARY

Fixes #3665 - allows lvols to be created with arguments prefixed with "+", to preserve old lvol.py behaviour

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

plugins/modules/system/lvol.py

### CODE REVIEW

File: changelogs/fragments/3681-lvol-fix-create.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - Fixes #3665 - allows lvols to be created with arguments prefixed with "+" to preserve old lvol.py behaviour
```
Comment (felixfontein) [inline changelogs/fragments/3681-lvol-fix-create.yml:2]:
```suggestion
  - lvol - allows logical volumes to be created with certain size arguments prefixed with ``+`` to preserve behavior of older versions of this module (https://github.com/ansible-collections/community.general/issues/3665).
```

**Comment by felixfontein** [comment]
@zigaSRC what do you think, could this work?

**Comment by zigaSRC** [comment]
@felixfontein Should be ok.

**Comment by felixfontein** [comment]
@jake2184 thanks for this improvement!
@zigaSRC thanks for reviewing!

**Pull request merged**